api/graphql/tests/test_reservations.py

```diff
@@ -1114,3 +1114,29 @@ def test_cancel_reservation_fails_when_reservation_in_past(self):
         assert_that(cancel_data.get("errors")).is_not_none()
         reservation.refresh_from_db()
         assert_that(reservation.state).is_equal_to(STATE_CHOICES.CONFIRMED)
+
+
+class ReservationQueryTestCase(ReservationTestCaseBase):
```

Comment:
Perhaps add a check that another user cannot see the reservee name.

Reply:
Added a test for this, which revealed that another user could've actually seen the reservee details. So I added new resolvers for each of the reservee fields and now they will return `None` if the user is not allowed to view those fields (e.g. user is different and not a superuser).